Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: build check test clippy compile_cairo clean
+	
+build: 
+	cargo build --release
+
+check: 
+	cargo check 
+
+compile_cairo:
```

Comment:
I think we can remove the syscalls.json file since we have a makefile target to generate it.

Reply:
yes! It was commited because we couldn't run cairo-compile in the CI, i just added a venv in order to do so and get the syscalls.json during the flow.